orchestrator/workflows/utils.py

```diff
@@ -322,6 +326,7 @@ def _validate_workflow(f: Callable[[], StepList]) -> Workflow:
             >> f()
             >> resync
             >> push_subscriptions(cache_domain_models)
+            >> refresh_subscription_search_index
```

Comment:
Curious to know what the impact of this will be on the engine, we used to have something similar to this, but we disabled it due to the performance cost (the database trigger). Even though it is not exactly the same, I'm a little concerned that this might be a bit too much. When running all validations in parallel, this will lock all tables way too often, imho and update the TSV, when there is no actual change to the subscription.

Reply:
I removed it